### PR TITLE
Run pre-commit to fix MaxText linter

### PR DESCRIPTION
**Pull request by bvandermoon**
# Description

CI lint tests are failing. This PR was auto-generated by `pre-commit run --all-files` to fix the lint issues.

# Tests

CI tests (auto-generated linter only)

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code. For an optional AI review, add the `gemini-review` label.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run end-to-end tests tests and provided workload links above if applicable.
- [x] I have made or will make corresponding changes to the doc if needed, including adding new documentation pages to the relevant Table of Contents (toctree directive) as explained in [our documentation](https://maxtext.readthedocs.io/en/latest/development.html#adding-new-documentation-files).

### CODE REVIEW

File: src/maxtext/layers/moe.py [hunk 1]
```diff
@@ -3333,10 +3285,7 @@ def fused_moe_matmul(
     It does not compute lb_loss or bias_updates (inference-only).
     """
     if forced_routed_experts is not None:
-      raise NotImplementedError(
-          "Forced routing via forced_routed_experts is not supported with"
-          " fused_moe_matmul."
-      )
+      raise NotImplementedError("Forced routing via forced_routed_experts is not supported with" " fused_moe_matmul.")
```
[the remark this answers is not in the document]
Comment (bvandermoon) [reply src/maxtext/layers/moe.py]: Fixed

File: tests/unit/router_replay_test.py [hunk 2]
```diff
@@ -568,28 +516,21 @@ def _assert_forced_routing_loss_finite(
 
     self.assertIsNotNone(loss)
     self.assertFalse(jnp.isnan(loss), "Loss must not be NaN")
-    print(
-        f"\n[Trainer Router Replay][{label}] Computed loss with forced routing"
-        f" + padding: {loss}"
-    )
+    print(f"\n[Trainer Router Replay][{label}] Computed loss with forced routing" f" + padding: {loss}")
```
[the remark this answers is not in the document]
Comment (bvandermoon) [reply tests/unit/router_replay_test.py]: Fixed

File: tests/unit/router_replay_test.py [hunk 3]
```diff
@@ -1010,10 +929,7 @@ def test_deepseek_scaling_is_not_renormalized_by_norm_topk_prob(self):
         float(jnp.sum(top_k_weights)),
         config.routed_scaling_factor,
         places=4,
-        msg=(
-            "DeepSeek weights must keep routed_scaling_factor; norm_topk_prob"
-            " must not re-normalize them."
-        ),
+        msg=("DeepSeek weights must keep routed_scaling_factor; norm_topk_prob" " must not re-normalize them."),
```
[the remark this answers is not in the document]
Comment (bvandermoon) [reply tests/unit/router_replay_test.py]: Fixed

File: tests/post_training/unit/router_replay_engine_test.py [hunk 4]
```diff
@@ -202,10 +194,7 @@ def test_engine_accepts_router_replay_payload_and_loss_is_finite(self):
         jnp.isnan(loss),
         "Loss must not be NaN when replaying router expert decisions",
     )
-    print(
-        "\n[Router Replay Engine] Computed loss with forced routing via"
-        f" MaxTextTrainingEngine: {loss}"
-    )
+    print("\n[Router Replay Engine] Computed loss with forced routing via" f" MaxTextTrainingEngine: {loss}")
```
[the remark this answers is not in the document]
Comment (bvandermoon) [reply tests/post_training/unit/router_replay_engine_test.py]: Fixed

**Review by NuojCheng — APPROVED** [review]
Thank you for the fix!

**Review by SurbhiJainUSC — APPROVED** [review]

**Review by darisoy — APPROVED** [review]
